shared/p2p/service.go

```diff
@@ -38,6 +38,8 @@ const prysmProtocolPrefix = "/prysm/0.0.0"
 // full beacon states over the wire for our current implementation.
 const maxMessageSize = 1 << 24
 
+const oneHour = time.Duration(1) * time.Hour
```

Comment:
This is only used once, please remove this const

Reply:
ok

Reply:
```suggestion
```